algorithms/CPlusPlus/Recursion/towerofHanoi.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+void towerOfHanoi(int disks,char stand1,char stand2,char stand3)
+{
+     if(disks==1)
+     {cout<<"Move disk 1 from "<<stand1<<" to "<<stand3<<endl;
+      return ;
+     }
+     towerOfHanoi(disks-1,stand1,stand3,stand2);
+     cout<<"Move disk "<<disks<<" from "<<stand1<<" to "<<stand3<<endl;
+     towerOfHanoi(disks-1,stand2,stand1,stand3);
+
```

Comment:
Correct the logical error in your recursion function. The code isn't working properly.

Reply:
For which test case (n=?) its wrong. I think code is absolutely correct.

Reply:
I ran this code on an online compiler. It didn't give any output.

Reply:
Run with some input for n on C++14 compiler.
[https://ide.geeksforgeeks.org/W2AEC8CCJI](url)

Reply:
[https://ide.geeksforgeeks.org/W2AEC8CCJI](url)

Reply:
Run with with some input for n (no of disks) on C++ 14 compiler.

https://ide.geeksforgeeks.org/W2AEC8CCJI

Reply:
Okay. Also, add the algorithm to the readme.MD file

Reply:
Done